CIP-0105/README.md

```diff
@@ -111,8 +111,8 @@ DRep keys and DRep IDs should be encoded in Bech32 with the following prefixes:
 | `drep_vk`     | CIP-1852’s DRep verification key                        | Ed25519 public key                                                |
 | `drep_xsk`    | CIP-1852’s DRep extended signing key                    | Ed25519-bip32 extended private key                                |
 | `drep_xvk`    | CIP-1852’s DRep extended verification key               | Ed25519 public key with chain code                                |
-| `drep`        | Delegate representative verification key hash (DRep ID) | blake2b\_224 digest of a delegate representative verification key |
-| `drep_script` | Delegate representative script hash (DRep ID)        | blake2b\_224 digest of a serialized delegate representative script |
+| `drep_vkh`    | Delegate representative verification key hash           | blake2b\_224 digest of a delegate representative verification key |
+| `drep_script` | Delegate representative script hash                     | blake2b\_224 digest of a serialized delegate representative script |
```

Comment:
```suggestion
| `drep` | [DEPRECATED] Delegate representative verification key hash (DRep ID) | blake2b\_224 digest of a delegate representative verification key |
| `drep_vkh`    | Delegate representative verification key hash           | blake2b\_224 digest of a delegate representative verification key |
| `drep_script` | Delegate representative script hash                     | blake2b\_224 digest of a serialized delegate representative script |
```

Reply:
Might be worth referencing CIP-129 where we say depreciated

Reply:
the deprecated tag links to the deprecation section